pipelines/ops/ml-mongo-dump/Jenkinsfile

```diff
@@ -0,0 +1,40 @@
+node() {
+    try {
+        String ANSI_GREEN = "\u001B[32m"
+        String ANSI_NORMAL = "\u001B[0m"
+        String ANSI_BOLD = "\u001B[1m"
+        String ANSI_RED = "\u001B[31m"
+        String ANSI_YELLOW = "\u001B[33m"
+
+        ansiColor('xterm') {
+            cleanWs()
```

Comment:
We dont need this and even the `stage('Checkout')` as we dont need the repo to be cloned.